content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx

```diff
@@ -0,0 +1,286 @@
+---
+title: Connection management
+metaTitle: Connection management (Guide)
+metaDescription: 'How to approach connection management with Prisma Client in serverless and PaaS applications.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+Databases can handle a limited number of concurrent connections. Each connection requires RAM, which means that simply increasing the connection limit without scaling the database:
+
+* ✔ might allow more processes to connect _but_
+* ✘ significantly affects database **performance**
+
+The way your application **manages connections** also impacts performance. This guide describes how to approach connection management in [serverless environments](#serverless-environments-faas) and [long-running processes](#long-running-processes-paas).
+
+</TopBlock>
+
+## Long-running processes (PaaS)
+
+Examples of long-running processes include Node.js applications hosted on a PaaS service like Heroku. Use the following checklist as a guide to connection management in PaaS environments:
+
+- ☐ Tune the [connection limit](#recommended-connection-limit)
```

Comment:
Is this about the connection limit of your database (max connection) or Prisma app instance (connection_limit)?